src/app/components/game/interface/menu-bar/home-register/chat-command-list/chat-command-list.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
why add this

Reply:
please review everything and check your comments for validity. This causes a lot of confusion in your coworkers